WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/BaseListUseCase.kt

```diff
@@ -121,6 +124,7 @@ class BaseListUseCase(
                                 }
                             }
                 }
+                mutableScrollTo.postValue(Event(visibleTypes.last()))
```

Comment:
We should probably check if we're refreshing before calling this, to prevent the screen from scrolling when we pull to refresh.

```suggestion
                if (!refresh) {
                    mutableScrollTo.postValue(Event(visibleTypes.last()))
                }
```